client/components/aboutUs/styles.js

```diff
@@ -0,0 +1,122 @@
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+import { Heading2, TextBody } from "../shared/styles";
+
+export const Wrapper = styled.div`
+  background: ${props => props.theme.colors.light};
+  padding: 30px 30px 50px;
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 55px 30px;
+  }
+`;
+
+export const SubInfo = styled.div`
```

Comment:
So the layout overall looks good! However on larger screens the icons and text drift apart quite a bit and feel disjointed (some items feel more left or right then centered.

That's just my opinion, seeing as the design does have a similar flow... I'll defer to @cMikolai and @Soreg for a better opinion on layout.

Reply:
Thank you for the feedback. I have made changes as per your feedback. Constraint the layout to stay in center.